pkg/3scale/amp/component/backend.go

```diff
@@ -2,16 +2,14 @@ package component
 
 import (
 	"fmt"
-	"strconv"
```

Comment:
Small but I would fix the imports order

Reply:
sure can do

Reply:
IDE keeps reverting these import orders , not a hill I am willing to die on.